ambari-server/src/main/java/org/apache/ambari/server/configuration/Configuration.java

```diff
@@ -5058,19 +5058,6 @@ public String getServerTempDir() {
     return getProperty(SERVER_TMP_DIR);
   }
 
-  /**
-   * Gets whether to use experiemental concurrent processing to convert
-   * {@link StageEntity} instances into {@link Stage} instances. The default is
-   * {@code false}.
-   *
-   * @return {code true} if the experimental feature is enabled, {@code false}
-   *         otherwise.
-   */
-  @Experimental(feature = ExperimentalFeature.PARALLEL_PROCESSING)
-  public boolean isExperimentalConcurrentStageProcessingEnabled() {
-    return Boolean.parseBoolean(getProperty(EXPERIMENTAL_CONCURRENCY_STAGE_PROCESSING_ENABLED));
```

Comment:
I think we can also remove `EXPERIMENTAL_CONCURRENCY_STAGE_PROCESSING_ENABLED` from `Configuration` and generate a new markup file as well.

Reply:
+1 for removing leftover property.

Reply:
Done